mis_builder/models/mis_report.py

```diff
@@ -176,14 +176,17 @@ def _onchange_name(self):
     def _compute_expression(self):
         for kpi in self:
             exprs = []
-            for expression in kpi.expression_ids:
-                if expression.subkpi_id:
-                    exprs.append(u'{}\xa0=\xa0{}'.format(
-                        expression.subkpi_id.name, expression.name))
-                else:
-                    exprs.append(
-                        expression.name or 'AccountingNone')
-            kpi.expression = ',\n'.join(exprs)
+            if kpi.expression_ids:
+                for expression in kpi.expression_ids:
+                    if expression.subkpi_id:
+                        exprs.append(u'{}\xa0=\xa0{}'.format(
+                            expression.subkpi_id.name, expression.name))
+                    else:
+                        exprs.append(
+                            expression.name or 'AccountingNone')
+                kpi.expression = ',\n'.join(exprs)
+            else:
+                pass
```

Comment:
Why changing this method?

Reply:
Sorry, we will revert those silly changes. These are leftovers of our tests to attempt to fix the problem described in the beginning.